trie/zk_trie_impl_test.go

```diff
@@ -9,18 +9,100 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	zktrie "github.com/scroll-tech/zktrie/trie"
+	zkt "github.com/scroll-tech/zktrie/types"
+
 	"github.com/scroll-tech/go-ethereum/common"
 	"github.com/scroll-tech/go-ethereum/core/types"
-	zkt "github.com/scroll-tech/go-ethereum/core/types/zktrie"
 	"github.com/scroll-tech/go-ethereum/ethdb/memorydb"
 )
 
+// we do not need zktrie impl anymore, only made a wrapper for adapting testing
```

Comment:
why not move this _test.go to https://github.com/scroll-tech/zktrie?

Reply:
good point, I will try later.

Reply:
Maybe not so convenience because: 
1. Many dependence to the modules inside go-ethereum in the test so it maybe too heavy to move it out to zktrie. 
2. Some other test in `zk_trie_proof` also use the `zkTrieImplTestWrapper` struct
3. Most important, the zktrie module itself is not "complete" (it require an external poseidon hash implement being provided) so it is a little trickly to test it inside the module itself. Currently we have had some "external" testings in the rust part.